test/unit/view.child-views.spec.js

```diff
@@ -300,7 +300,7 @@ describe('layoutView', function() {
       var ChildView = Marionette.View.extend({
         template: false,
         onRender: function() {
-          this.triggerMethod('content:rendered');
```

Comment:
@scott-w would you prefer another childViewEvents https://github.com/marionettejs/backbone.marionette/pull/3247/files#diff-0f13482af07d71e12b54fc8c40be7af6L293 to cover the `triggerMethod` case?